src/scripts/modules/media/body/body.coffee

```diff
@@ -152,6 +152,31 @@ define (require) ->
             # NOTE: `.toggleClass()` explicitly requires a `false` (not falsy) 2nd argument
             $el.toggleClass('ui-has-child-title', $title.length > 0)
 
+          # Create subtitles in each chapter with corresponding ids as their href
```

Comment:
Well, I created a simple table of ocntents for the individual page view, which is basically a <ol>, Is that enough? Or do I need to make it almost same as the table of contents in normal page?